hummingbird/ml/_topology.py

```diff
@@ -9,6 +9,7 @@
 """
 import numpy as np
 import os
+from packaging import version
```

Comment:
A super minor thing. Throughout Hummingbird we use `from distutils.version import LooseVersion` (instead of this) and then we do for example `LooseVersion(torch.__version__) <= LooseVersion("1.4")`. Do you think you can use the same approach? Just for consistency really.

Reply:
Of course. Done.